src/cashlib/cashlib.cpp

```diff
@@ -407,7 +407,9 @@ SLAPI void *CreateScriptMachine(unsigned int flags,
 
     // Its ok to get the bare tx pointer: the life of the CTransaction is the same as TransactionSignatureChecker
     smd->checker = std::make_shared<TransactionSignatureChecker>(smd->tx.get(), inputIdx, inputAmount, flags);
-    smd->sm = new ScriptMachine(flags, *smd->checker, 0xffffffff);
+    // max ops is set the maximum value with the intention that the caller will check these if needed
```

Comment:
spelling?
-// max ops is set the
+//max ops is set to the

Also, should it be max ops or max sigchecks?

Reply:
done